Chandra/cmd_states/cmd_states.py

```diff
@@ -101,12 +102,36 @@ def decode_power(mnem):
 
 def _make_add_trans(transitions, date, exclude):
     def add_trans(date=date, **kwargs):
-        # if no key in kwargs is in the exclude set then update transition
+        # If no key in kwargs is in the exclude set then update transition
+        # And, when doing any update, update a bookkeeping key 'last_date'
+        # that stores the latest date of any processed cmd/transition.
+        # This is used to prevent overlap between the GET_PITCH mocked
+        # up cmds and the inserted/mock maneuver cmds.  If any one of the
+        # equally-spaced GET_PITCH cmds occurs during a maneuver, the GET_PITCH
+        # will be ignored in processing because the maneuver cmd insertion
+        # through the maneuver time range will have updated last_date to
+        # a time later than that GET_PITCH command.
         if not (exclude and set(exclude).intersection(kwargs)):
             transitions.setdefault(date, {}).update(kwargs)
+            # Only update 'last_date' if the supplied date to _make_add_trans
+            # is actually later than the stored 'last_date' in the structure
+            if date > transitions['last_date']:
+                transitions['last_date'] = date
     return add_trans
 
 
+def _make_pitch_cmds(datestart, datestop, sample_time=10000.):
+    """
+    Add cmds to clock out pitch for any long states
+    """
+    return [{'cmd': 'GET_PITCH',
+             'tlmsid': 'GET_PITCH',
+             'date': DateTime(t).date}
+            for t in np.arange(np.floor(DateTime(datestart).secs / sample_time),
```

Comment:
And a comment in the docstring or code explaining the motivations for the sample_time quantization would be useful.